distributed/FSDP2/run_example.sh

```diff
@@ -0,0 +1,11 @@
+# /bin/bash
+# bash run_example.sh {file_to_run.py} {num_gpus}
+# where file_to_run = example to run. Default = 'example.py'
+# num_gpus = num local gpus to use (must be at least 2). Default = 4
+
+# samples to run include:
+# example.py
+
+echo "Launching ${1:-example.py} with ${2:-4} gpus"
+torchrun --nnodes=2 --nproc_per_node=${2:-4} ${1:-example.py}
```

Comment:
```suggestion
echo "Launching ${1:-example.py} with ${2:-2} gpus"
torchrun ---nproc_per_node=${2:2} ${1:-example.py}
```


- you are asking for 2 gpu in the `verify_min_gpu_count` function. 
- remove `--nnodes=2`  that is for multi node training. That is not possible with the current CI